### PR TITLE
#304 Add /user/all endpoint. Turn off permission logs

### PR DESCRIPTION
**Pull request by HenoJurek**
- Add /user/all endpoint that return list of authorized admin events users.
   New endpoint return only users that take part in same events that admin(actualy login admin)
- Update swagger docu.
- Turn off permission logs.

### CODE REVIEW

File: controllers/user/all.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    database.readMany('users', onlyAdminEventsUsersQuery)
+      .then(results => {
+        const filteredUsersList = results.map(userObject => ({
+          user: userObject.user,
+          userTeam: userObject.userTeam,
+          accountIsActive: userObject.accountIsActive,
+          accountCreated: userObject.accountCreated,
+          collectedPointsIds: userObject.collectedPointsIds,
+        }));
+        res.send(filteredUsersList);
```
Comment (dbetka) [inline controllers/user/all.js:62]: Front-end always gets response with `error` field.
I think response should be like below:
```
{
  users: [...],
  error: null
}
```

**Review by dbetka — CHANGES_REQUESTED** [review]
To change

**Comment by dbetka** [comment]
I've added front-end side functionality (API)

File: controllers/user/all.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+    database.readMany('users', onlyAdminEventsUsersQuery)
+      .then(results => {
+        const filteredUsersList = results.map(userObject => ({
+          user: userObject.user,
+          userTeam: userObject.userTeam,
+          accountIsActive: userObject.accountIsActive,
+          accountCreated: userObject.accountCreated,
+          collectedPointsIds: userObject.collectedPointsIds,
+        }));
```
Comment (dbetka) [inline controllers/user/all.js:61]: I also need an accountType

**Review by dbetka — CHANGES_REQUESTED** [review]

File: controllers/user/all.js (hunk 1 above)
Comment (HenoJurek) [reply controllers/user/all.js:62]: fixed

File: controllers/user/all.js (hunk 2 above)
Comment (HenoJurek) [reply controllers/user/all.js:61]: added

**Pull request merged**